index.html

```diff
@@ -33,5 +33,99 @@ <h2 class="connect">LET'S CONNECT</h2>
                 <li><a href="https://angel.co/u/dany-augustin"><img src="img/angellist.svg" alt="angellist-icon"></a></li>
             </ul>
     </section>
+
+<section class="Grid-boxes">
+    <article class="Box">
+        <img class="box-image" src="img/tonic.png" alt="tonic">
+        <div class="content">
+            <h2 class="content-title">1-Tonic</h2>
+            <div class="text-title">
+                <a href="#" class="client">CANOPY</a>
+                <img class="dot" src="img/dot.png" alt="dot">
+                <a href="#" class="client-skills">BACK END DEV</a>
+                <img class="dot" src="img/dot.png" alt="dot">
+                <a href="#" class="period">2015</a>
+            </div>
+            <p class="content-text">A daily selection Of privately personalized reads; no accounts or sign-ups required
+            </p>
+            <ul class="tags">
+                <li class="style">html</li>
+                <li class="style">css</li>
+                <li class="style">javascript</li>
+            </ul>
+            <a href="#" class="link">See Project</a>
+        </div>
+    </article>
+    <article class="Box">
+        <img class="box-image" src="img/multi.png" alt="multi">
+        <div class="content">
+            <h2 class="content-title">2 Multi-Post Stories</h2>
+            <div class="text-title">
+                <a href="#" class="client">CANOPY</a>
+                <img class="dot" src="img/dot.png" alt="dot">
+                <a href="#" class="client-skills">BACK END DEV</a>
+                <img class="dot" src="img/dot.png" alt="dot">
+                <a href="#" class="period">2015</a>
+            </div>
+            <p class="content-text">A daily selection Of privately personalized reads; no accounts or sign-ups required
+            </p>
+            <ul class="tags">
+                <li class="style">html</li>
+                <li class="style">css</li>
+                <li class="style">javascript</li>
+            </ul>
+            <a href="#" class="link">See Project</a>
+        </div>
+    </article>
+</section>
+
+<section class="About-container">
+    <div class="top">
+        <h2 class="about-head">About </br> Myself</h2>
+        <p class="text">Hello I’m a software developer! I can help you build a product , feature or website Look through
+            some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate
+            to contact me.</p>
+        <h3 class="helpingtext">LET'S CONNECT</h3>
+        <ul class="belowtext">
```

Comment:
The about myself and the project sections are overlapping. Kindly adjust the design so theatthey don't overlap.

Your design:
![image](https://user-images.githubusercontent.com/20554583/219922445-f98ac962-1c7a-43e5-a465-a6451134ba72.png)

Figma design:
![image](https://user-images.githubusercontent.com/20554583/219922518-f7582ca1-ea43-4dcc-addb-b44997b16baf.png)

Please, also take note of the background that separates them and the curve at the top right edge of the about section.